src/EnergyPlus/EconomicLifeCycleCost.hh

```diff
@@ -96,26 +109,155 @@ namespace EconomicLifeCycleCost {
         Straight39,
         Straight40,
         None,
+        Num
+    };
+
+    int constexpr SizeDepr(41);
+
+    constexpr std::array<std::array<Real64, SizeDepr>, static_cast<int>(DeprMethod::Num)> DepreciationPercentTable{
```

Comment:
This isn't the easiest thing to read/maintain, but I think it's a net gain.  The huge IF block, non-constexpr, assignments weren't great either.  Maybe there is a better way to format the tabular data ... I dunno.  No need to address it now.

Reply:
Would it be better to split stuff up to improve readability? Something like,
```
constexpr std::array<string_view> smallTable1;
constexpr std::array<string_view> smallTable2;

constexpr std::array<std:array> bigTable {smallTable1, smallTable2};
```